src/airgap/upload_all_bundles.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+upload_to_bucket() {
+    export GOOGLE_APPLICATION_CREDENTIALS=$1
```

Comment:
Why do we require this here? It is not being used anyway here right?

Reply:
this will use command line arg that we are passing service account details.